habitat_baselines/config/default.py

```diff
@@ -14,92 +14,83 @@
 DEFAULT_CONFIG_DIR = "configs/"
 CONFIG_FILE_SEPARATOR = ","
 # -----------------------------------------------------------------------------
-# Config definition
+# EXPERIMENT CONFIG
 # -----------------------------------------------------------------------------
 _C = CN()
 _C.BASE_TASK_CONFIG_PATH = "configs/tasks/pointnav.yaml"
 _C.TASK_CONFIG = CN()  # task_config will be stored as a config node
 _C.CMD_TRAILING_OPTS = ""  # store command line options"
```

Comment:
Why do we store command lines in config, not just merge them in config when passed? That can create confusion.

Reply:
This is to support the ideal overwrite behavior when loading saved config upon evaluation run. See [here](https://github.com/JasonJiazhiZhang/habitat-api/blob/743e82e764e2a8b29dee1c93ef54815b7d192b03/habitat_baselines/rl/ppo/ppo_trainer.py#L344-L354).